create-cluster/src/index.ts

```diff
@@ -10,7 +10,7 @@ async function run() {
     const k8sDistribution = core.getInput('kubernetes-distribution');
     const k8sVersion = core.getInput('kubernetes-version');
     const k8sTTL = core.getInput('ttl');
```

Comment:
How does TTL get it's default vs how timeout-minutes gets a default? They should be the same IMO

Reply:
`timeout-minutes` is not an exposed input and has no default from the `action.yaml`.
I'll add it also for ttl, but it will be redundant, as ttl is exposed and has a default.